extensions.tf

```diff
@@ -1,7 +1,13 @@
 
 module "vm-bootstrap" {
+  providers = {
+    azurerm     = azurerm
+    azurerm.cnp = azurerm.cnp
+    azurerm.soc = azurerm.soc
+  }
+
   count  = var.install_splunk_uf == true || var.nessus_install == true ? 1 : 0
-  source = "git::https://github.com/hmcts/terraform-module-vm-bootstrap.git?ref=master"
+  source = "git::https://github.com/hmcts/terraform-module-vm-bootstrap.git?ref=DTSPO-14772"
```

Comment:
is this just for testing? we want to keep `master` consistent

Reply:
yes was for testing until it works. changed back to master now